src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -936,24 +937,6 @@ class DevTask extends AbstractServerTask {
                 setDockerBuildTimeout(buildDockerBuildTimeoutSetting);
             }
         }
-
-        // set container param if dockerfile or dockerRunOpts are set
-        if (!container) {
-            if (dockerfile != null) {
-                if (dockerfile.exists()) {
-                    setContainer(true);
-                    return;
-                } else {
-                    throw new Exception("The file " + dockerfile + " used for dev mode option dockerfile does not exist."
```

Comment:
Do we perform this check anywhere else to see if the Dockerfile exists?

Reply:
This is checked in ci.common devutil.java line 560.